### DISCUSSION TITLE
(CANT BOOT) ucsi_gg_init failed error on Nvidia GPU

### DISCUSSION THREAD

**Discussion by fn1y** [elementary/triage#523] [Q&A]
Hey guys. New to Linux. Was trying to get my nvidia graphics card set up and was getting an error on boot complaining about ‘ucsi_gg’. After some googling I found out that it’s an error where the driver tries to load a usb c port on the gpu (mine doesn’t have one!) so followed someone’s advice to add a file at /etc/modprobe.d/blacklist-nvidia-usb.conf with the string blacklist ucsi_ccg.

I did this, and my computer instead just booted to a blinking cursor. So in recovery mode, I removed the new blacklist file and I'm back to square one. My computer doesn't boot unless it's recovery mode. I've attached a pic of the error!

I’m using an Nvidia RTX 2060 and installed the nvidia-535 drivers through apt in the terminal, and they show up as properly installed in the updates section of the pre-installed App Store

![IMG_2732](https://github.com/elementary/triage/assets/61245581/1bd71d4f-40b2-4a76-a41f-52ff21a26554)

**Comment by T-Atlas** [comment]
Have you resolved it? I encountered the same issue as well.
